orchestrator/services/subscriptions.py

```diff
@@ -115,7 +115,7 @@ def update_subscription_status(subscription_id: UUIDstr, status: str) -> Subscri
 
 
 def update_subscription_description(subscription_id: UUIDstr, description: str) -> SubscriptionTable:
-    """Update a subscriptions description.
+    """Update a subscriptions' description.
```

Comment:
```suggestion
    """Update a subscription's description.
```

If we're going to put salt on all snails, I think it should be like this :)